src/datachain/query/dataset.py

```diff
@@ -876,12 +931,18 @@ def apply_sql_clause(self, query: Select) -> Select:
 class SQLOffset(SQLClause):
     offset: int
 
+    def hash_inputs(self) -> str:
+        return hashlib.sha256(str(self.offset).encode()).hexdigest()
```

Comment:
move `hashlib.sha256(str(self.offset).encode()).hexdigest()` into a helper? it will make code cleaner a lot ... since it repeats so many times